src/ui/public/vis/agg_config_result.js

```diff
@@ -3,7 +3,7 @@ define(function () {
 
   function AggConfigResult(aggConfig, parent, value, key) {
     this.key = key;
-    this.value = value;
+    this.value = value === null ? false : value; // restrict to primitive for sorting in Angular
```

Comment:
I agree, `null` is perfectly valid value for `AggConfigResult, the only problem is with sorting so we should fix it there.